README.md

```diff
@@ -54,6 +54,38 @@ NET_RECEIVE(weight (uS)) {
 
 See [INSTALL.md](https://github.com/BlueBrain/nmodl/blob/master/INSTALL.md) for detailed instructions to build the NMODL from source.
 
+### Taking NMODL for a spin
+
+To quickly test the NMODL Framework's analysis capabilities we provide a
+[docker](https://www.docker.com) image, which includes the NMODL Framework python library and a
+fully functional Jupyter notebook environment.
+
+After installing docker you can pull and run the NMODL image from your terminal:
+
+```sh
+docker run -it -p 8888:8888 -v ~/yourpath/to/nmodl/notebooks:/nmodl/notebooks ohm314/nmodl:test
```

Comment:
With the current command, if the user creates a Jupyter notebook from the container, the file will belong to *root* on the container.
You can quickly review the 2nd tech-talk I made about Docker.
https://github.com/CNS-OIST/STEPS_Docker/ provide a possible solution for this issue.

Reply:
thanks, I improved that a bit :)